engines/bladerunner/actor.cpp

```diff
@@ -40,6 +40,9 @@
 #include "bladerunner/set.h"
 #include "bladerunner/slice_animations.h"
 #include "bladerunner/slice_renderer.h"
+#if SUBTITLES_SUPPORT
```

Comment:
Why do you make it optional compilation at all?

Reply:
I'm using multiple such defines on another local repository where I'm working on restoring dialogue and recreating some "deleted" scenes. These defines help quickly locate where changes for a relevant feature were and disable them if something malfunctions, so I kept them here. I agree that they are not essential of course.

Reply:
It would be nice to remove them for ScummVM, as we prefer to have as less global defines as possible